### ISSUE TITLE
Facing the “Certificate ARN is not valid” issue in Clara Train SDK configuration

### ISSUE THREAD

**Issue by SomasekharDS** [aws-quickstart/quickstart-nvidia-imaging-clara-train#10]
Hi Team,

Thanks for the Awesome AWS quick starter guide for Clara Train SDK.

I am trying to configure the Clara Train SDK in my personal AWS account before proceeding to configure it in my organization account. I have gone through the Clara AWS deployment document.

https://s3.amazonaws.com/aws-quickstart/quickstart-nvidia-imaging-clara-train/doc/nvidia-clara-train-on-the-aws-cloud.pdf

On the configuration process of Clara Train SDK, I am facing the “Certificate ARN is not valid” issue. Please find the below screenshots.

![image](https://user-images.githubusercontent.com/24861755/91430853-10073b00-e892-11ea-8c9d-129865146259.png)

Kindly look into the issue and provide a fix for this issue.

Appreciate your help.

Thank you.

**Comment by ajfriedman18** [comment]
Did you enter in an ARN or leave the field blank?

Please don't paste in the account ID, but if you could let us know whether you had a parameter mapping to an existing cert or left it blank that would be helpful.

**Comment by SomasekharDS** [comment]
Hello Peer,

Many thanks for your reply.. you are right I have left blank for the field ARN, so I have modified the "UseHTTPS". Now i am not facing Certificate ARN issue, But when I move forward in the configuration process of Clara Train SDK, I am facing the “ECSAutoScalingGroup creation time out” issue.  So for that particular issue, I have created a new issue and requested the community for help.  If you have some time kindly provide your valuable pointers for resolution. 

https://github.com/aws-quickstart/quickstart-nvidia-imaging-clara-train/issues/11

Once again many thanks for your reply.
